R/combine_agent_data.R

```diff
@@ -0,0 +1,106 @@
+library("tidyverse")
+library("config")
+library("here")
+
+# Helper functions ----
+
+# Read files wrapper for agent data
+read <- function(name) {
+  read_delim(file.path(config::get("helmet_data"),
+                       config::get(name),
+                       "agents.txt"),
+             delim = "\t",
+             col_names = TRUE) %>%
+    select(-"X1")
+}
+
+# Grouping wrapper for agent data
+group <- function(df, group_var) {
+  df %>%
+    mutate(persons = 1) %>%
+    group_by(!!!syms(group_var)) %>%
+    summarise_all(sum, na.rm = TRUE) %>%
```

Comment:
`dplyr::summarise_all` will be deprecated in favor of `dplyr::across`. Could you modify this?